pkg/agent/openflow/pipeline.go

```diff
@@ -1342,11 +1352,6 @@ func (c *client) addFlowMatch(fb binding.FlowBuilder, matchKey *types.MatchKey,
 		fb = fb.MatchProtocol(matchKey.GetOFProtocol()).MatchSrcIPNet(matchValue.(net.IPNet))
 	case MatchSrcIPNetv6:
 		fb = fb.MatchProtocol(matchKey.GetOFProtocol()).MatchSrcIPNet(matchValue.(net.IPNet))
-	case MatchDstOFPort:
-		// ofport number in NXM_NX_REG1 is used in ingress rule to match packets sent to local Pod.
-		fb = fb.MatchProtocol(matchKey.GetOFProtocol()).MatchReg(int(PortCacheReg), uint32(matchValue.(int32)))
-	case MatchSrcOFPort:
-		fb = fb.MatchProtocol(matchKey.GetOFProtocol()).MatchInPort(uint32(matchValue.(int32)))
```

Comment:
these switch cases match earlier ones, my understanding is that they are therefore never used (no fallthrough)